app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -60,6 +61,11 @@ def collect_clusters
       connection.system_service.clusters_service.list
     end
 
+    def collect_cluster(uuid)
+      cluster = connection.system_service.clusters_service.cluster_service(uuid).get
+      [cluster]
```

Comment:
Parser code expects to have an array and it is consistent with how we do it in other places.